src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
```

Comment:
box-sizing should be the same for all elements, move it to `*` selector rule in normalize